packages/tradershub/src/modals/AddOrManageAccount/AddOrManageAccount.tsx

```diff
@@ -37,6 +37,11 @@ const AddOrManageAccount = ({ isOpen, onClose }: TAddOrManageAccount) => {
         }
     }, [isEU]);
 
+    const addedAllCryptoCurrencies = useMemo(
+        () => currencies?.CRYPTO.every(currency => currency.isAdded) ?? false,
+        [currencies]
```

Comment:
Can we move this logic to the useCurrencies hook, reduce complexity of the component